static/index.js

```diff
@@ -0,0 +1,82 @@
+let reviews = [];
+
+let sum = 0;
+
+if(localStorage.getItem("reviews") !== null) reviews = JSON.parse(localStorage.getItem("reviews"));
+
+
+
+const updateAverage = () =>{
+    document.querySelector("#averageRating").innerText = `${Math.round(sum / reviews.length * 10) / 10}/5`;
+}
+
+const createItem = (item) => {
+    let reviewItem = document.createElement("div");
+    let reviewText = document.createElement("p");
+    reviewText.innerText = item.review;
+    
+    let ratingText = document.createElement("p");
+    ratingText.innerText = `${item.rating} / 5`;
+
+    let deleteButton = document.createElement("button");
+    deleteButton.innerText = "Delete";
+    deleteButton.id = `item-${item.id}`;
```

Comment:
It would be smart for the button id to be more descriptive than just the id of the todo. Something like `button-delete-{id}`.

Reply:
Fixed in [Improvements](https://github.com/infinum-academy/js-react-academy-2024-filiptodosov/pull/1/commits/79575e604ff61d5206a96a781ad054151af9c333)